prac_06/languages.py

```diff
@@ -0,0 +1,13 @@
+from prac_06.programming_language import ProgrammingLanguage
+
+python = ProgrammingLanguage("Python", "Dynamic", True, 1991)
+ruby = ProgrammingLanguage("Ruby", "Dynamic", True, 1995)
+visual_basic = ProgrammingLanguage("Visual Basic", "Static", False, 1991)
+# print(python)
+# print(ruby)
+# print(visual_basic)
+languages = [python, ruby, visual_basic]
+print(f"The dynamically typed languages are:")
+for language in languages:
```

Comment:
I think this code is good enough but I do think it would be better to give the language(s) a specific name. For example, dynamic language or coding language. Just a suggestion tho but it's up to you.

Reply:
Noted. Thank you.